portal-web/test/functional/com/liferay/portalweb/macros/site/Site.macro

```diff
@@ -1318,6 +1318,8 @@ definition {
 		else {
 			Alert.viewErrorMessage(errorMessage = "The Site Name cannot be blank, numeric or a reserved word such as null. The Site Name cannot contain the following invalid characters: *.");
 		}
+
+		SelectFrameTop();
```

Comment:
Based on the commit message I would have thought this SelectFrameTop() would be called before the if/else statement

Reply:
Based on failing poshi report [here](https://testray.liferay.com/home/-/testray/case_results/1912831636?redirect=https%3A%2F%2Ftestray.liferay.com%2Fhome%2F-%2Ftestray%2Fcase_results%2Findex%3FtestrayCaseName%3DCanNotAddSiteThatDuplicatesName%26testrayBuildId%3D1912820045%26cur%3D1%26delta%3D200%26orderByCol%3Dstatus_sortable%26orderByType%3Dasc), I would have imagined just removing the SelectFrame on line 1313 would work since the call is duplicated in Button.add